src/DynamoCoreWpf/Windows/ExtensionWindow.xaml

```diff
@@ -7,6 +7,8 @@
                            xmlns:ui="clr-namespace:Dynamo.UI"
                            mc:Ignorable="d" 
                            d:DesignHeight="450" d:DesignWidth="800"
+                           MinHeight="32"
+                           MinWidth="160"
```

Comment:
Would you attach an example of the extension window with minimal size to this PR so we have a record?

Reply:
That's only big enough so to make the title bar not disappear.